challenge/src/mandalorian/BabyYoda.css

```diff
@@ -16,6 +16,7 @@
 
 .baby-yoda h1 {
     font-family: Impact, sans-serif;
+    font-size: 72px;
```

Comment:
this messed up the alignment on the baby yoda component. now the image is still left aligned but the text is longer so it looks a little wonky. maybe try going into babyyoda.jsx and see if you can center that image

Reply:
Made the changes in the latest commit